v20.2/create-table-as.md

```diff
@@ -79,6 +85,8 @@ table td:first-child {
  `create_as_constraint_def` | An optional [primary key constraint](primary-key.html).
  `select_stmt` | A [selection query](selection-queries.html) to provide the data.
  `opt_persistence_temp_table` |  Defines the table as a session-scoped temporary table. For more information, see [Temporary Tables](temporary-tables.html).<br><br>Note that the `LOCAL`, `GLOBAL`, and `UNLOGGED` options are no-ops, allowed by the parser for PostgresSQL compatibility.<br><br>**Support for temporary tables is [experimental](experimental-features.html#temporary-objects)**.
+ `opt_with_storage_parameter_list` | This clause is a no-op, allowed by the parser for PostgresSQL compatibility. Allowed parameters include `fillfactor`, `s2_max_level`, `s2_level_mod`, `s2_max_cells`, `geometry_min_x`, `geometry_max_x`, `geometry_min_y`, and `geometry_max_y`.
```

Comment:
not entirely a no-op -- the `s2_*` commands configure geospatial indexes

Reply:
Oh wow you are totally right. Updated this.